src/state.rs

```diff
@@ -1,6 +1,7 @@
 use super::memory::MemoryManagers;
 use crate::game_manager::GameManager;
 use crate::gui::Gui;
+// use puffin_egui::puffin;
```

Comment:
Bit confused about the `puffin` changes overall, since all of it seems to be commented out.

Reply:
I made some adjustments prior to fixing the fps thing, but included them here anyways. 
I still need to set up a custom profile for puffin so you'd do `cargo run --puffin` 

You can ignore these as a TODO